priv/template/config/dev.exs

```diff
@@ -3,7 +3,12 @@ use Mix.Config
 config :<%= application_name %>, <%= application_module %>.Endpoint,
   http: [port: System.get_env("PORT") || 4000],
   debug_errors: true,
-  cache_static_lookup: false
+  cache_static_lookup: false,
+  watchers: [{Path.expand("node_modules/brunch/bin/brunch"), ["watch"]}],
+  live_reload: ["priv/static/app.js",
+                "priv/static/app.css",
+                Path.wildcard("web/templates/**/*")]
```

Comment:
I think we should include template watching by default, but add a comment along the lines of:

```
  # Watch assets and templates for live-reload changes
  # Note: larger projects will use higher CPU in dev as the number of files grows
```